src/org-manager/containers/organisation-details/organisation-details.component.ts

```diff
@@ -20,14 +20,28 @@ export class OrganisationDetailsComponent implements OnInit {
   constructor(private store: Store<fromStore.OrganisationRootState>) {}
 
   public ngOnInit(): void {
-   this.store.pipe(select(fromStore.getAllLoaded)).pipe(takeWhile(loaded => !loaded)).subscribe(loaded => {
+    this.store.pipe(select(fromStore.getAllLoaded)).pipe(takeWhile(loaded => !loaded)).subscribe(loaded => {
       if (!loaded) {
         this.store.dispatch(new fromOrganisation.LoadActiveOrganisation());
         this.store.dispatch(new fromOrganisation.LoadPendingOrganisations());
       }
     });
 
-   this.orgs$ = this.store.pipe(select(fromStore.getActiveAndPending));
+    this.orgs$ = this.store.pipe(select(fromStore.getActiveAndPending));
+    this.orgs$.pipe(
+        filter(value => value !== undefined),
+        take(1)
+    ).subscribe(({organisationId, pbaNumber, isAccLoaded}) => {
+      if (!isAccLoaded && pbaNumber.length) {
+        this.store.dispatch(new fromOrganisation.LoadPbaAccountsDetails({
+              orgId: organisationId,
+              pbas: pbaNumber.toString()
```

Comment:
pbaNumber is already string. Do we need toString() method?

Reply:
pba in the state are stored as array so at this point i am making sting out of it and sending it to effects -> services